BUILDING.md

```diff
@@ -139,17 +138,16 @@ support on x86_64 and ia32.
 * llvm version 3.3 or higher
 * nasm version 2.10 or higher in Windows
 
-Otherwise `configure` will fail with an error. This can be avoided by
-either providing a newer assembler as per the list above or by
-using the `--openssl-no-asm` flag.
+Otherwise `configure` will fail with an error. Avoid this by either providing a
```

Comment:
```suggestion
To avoid `configure` failing in case this requirement is not met, run with `--openssl-no-asm`.
```

Reply:
How about this?: 

```markdown
To use an earlier assembler version, use the `--openssl-no-asm` flag with
`configure`. Otherwise, `configure` will fail.
```

Reply:
@Trott `--openssl-no-asm` skips the use of assembler, so it would not be using earlier versions.

Reply:
OK, how about this?:

```markdown
If compiling without one of the above, use `configure` with the
`--openssl-no-asm` flag. Otherwise, `configure` will fail.
```